app/assets/javascripts/discourse/tests/acceptance/invite-accept-test.js

```diff
@@ -8,6 +8,43 @@ import PreloadStore from "discourse/lib/preload-store";
 import I18n from "I18n";
 import { test } from "qunit";
 
+function setAuthenticationData(hooks, json) {
+  hooks.beforeEach(() => {
+    const node = document.createElement("meta");
+    node.dataset.authenticationData = JSON.stringify(json);
+    node.id = "data-authentication";
+    document.querySelector("head").appendChild(node);
+  });
+  hooks.afterEach(() => {
+    document
+      .querySelector("head")
+      .removeChild(document.getElementById("data-authentication"));
+  });
+}
+
+function preloadInvite({ link } = { link: false }) {
```

Comment:
🏌️⛳ (though functionally it's not an exact equivalent 😉)

```suggestion
function preloadInvite({ link = false } = {}) {
```